src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml

```diff
@@ -0,0 +1,1194 @@
+                                                       Click="FlyoutButton_Clicked"
+                                                       Content="&#xE94E;"
+                                                       IsEnabled="{x:Bind InequalityButton.IsEnabled, Mode=OneWay}"/>
+
+                            <controls:CalculatorButton x:Name="GreaterThanOrEqualFlyoutButton"
+                                                       x:Uid="greaterThanOrEqualFlyoutButton"
+                                                       Grid.Column="3"
+                                                       Style="{StaticResource SymbolOperatorButtonStyle}"
+                                                       AutomationProperties.AutomationId="greaterThanOrEqualFlyoutButton"
+                                                       Click="FlyoutButton_Clicked"
+                                                       Content="&#xF88D;"
+                                                       IsEnabled="{x:Bind InequalityButton.IsEnabled, Mode=OneWay}"/>
+
+                            <controls:CalculatorButton x:Name="GreaterThanFlyoutButton"
+                                                       x:Uid="greaterThanFlyoutButton"
+                                                       Grid.Column="4"
+                                                       Style="{StaticResource SymbolOperatorButtonStyle}"
+                                                       AutomationProperties.AutomationId="greaterThanFlyoutButton"
+                                                       Click="FlyoutButton_Clicked"
+                                                       Content="&#xF88C;"
+                                                       IsEnabled="{x:Bind InequalityButton.IsEnabled, Mode=OneWay}"/>
+
+                        </Grid>
+                    </Flyout>
+                </controls:OperatorPanelButton.FlyoutMenu>
+            </controls:OperatorPanelButton>
+
+            <controls:OperatorPanelButton x:Name="FuncButton"
+                                          x:Uid="funcButton"
+                                          Style="{StaticResource OperatorPanelButtonStyle}"
+                                          AutomationProperties.AutomationId="funcButton"
+                                          Glyph="&#xF893;">
+
+                <controls:OperatorPanelButton.FlyoutMenu>
+                    <Flyout x:Name="FuncFlyout"
+                            Windows10version1809:AreOpenCloseAnimationsEnabled="False"
+                            FlyoutPresenterStyle="{ThemeResource OperatorPanelFlyoutStyle}"
+                            Placement="Bottom">
+
+                        <Grid x:Name="FuncGrid"
+                              MinWidth="194"
+                              MinHeight="48"
+                              XYFocusKeyboardNavigation="Enabled">
+                            <Grid.RowDefinitions>
```

Comment:
What should we do when this one is clicked?  Do we add double bars and set the cursor between the two?

Reply:
That would work, if we want to be consistent with other buttons like this then abs(-6) also works

Reply:
actually, this is probably not a conversation we need to have here as it isn't super relevant to the code at hand.